packages/openapi2-parser/lib/schema.js

```diff
@@ -203,11 +203,6 @@ class DataStructureGenerator {
           return uniqueTypes[0];
         }
       }
-
-      if (schema.properties) {
```

Comment:
this removes only assumption we made right?

Reply:
That's right